src/kontrol/__main__.py

```diff
@@ -492,7 +492,11 @@ def _parse_test_version_tuple(value: str) -> tuple[str, int | None]:
         dest='tests',
         default=[],
         action='append',
-        help='Limit to only listed tests, ContractName.TestName',
+        help=(
+            "Specify the contract method to test in the format 'ContractName.MethodName'. If a method is overloaded, "
```

Comment:
One minor comment I have here is that I usually see “functions” instead of “methods” in the context of Solidity smart contracts. I agree that it doesn’t seem right, cause, unlike methods, functions are typically not associated with objects (or something like this), but, in my experience, methods are usually referred to as “Solidity functions” most of the time — probably because of the `function` keyword used in the definition. That doesn't have to addressed, just a suggestion to consider.

Reply:
Addressed this, thank you! I got a bit confused because we're using both `method` and `function` keywords in the codebase. Left `function` to align with all the other help messages.